custom_components/localtuya/pytuya/__init__.py

```diff
@@ -216,6 +216,61 @@ def _send_receive(self, payload):
         s.close()
         return data
 
+    def _decode_received_data(self, data, is_status):
```

Comment:
As I mentioned, I'm cleaning up pytuya quite a bit and would greatly appreciate if we wait with these changes until I'm done. It will be a lot easier to solve this conflict than doing it on my end.

Reply:
OK no problem, let's wait for your part to be ready.